runtime/doc/eval.txt

```diff
@@ -8927,6 +8927,18 @@ settagstack({nr}, {dict} [, {action}])			*settagstack()*
 			let newtag = [{'tagname' : 'mytag', 'from' : pos}]
 			call settagstack(2, {'items' : newtag}, 'a')
 
+<		    Write a jump to the tag stack: >
+			" Store where we're jumping from.
+			let pos = [bufnr()] + getcurpos()[1:]
+			let item = {'bufnr': pos[0], 'from': pos, 'tagname': expand('<cword>')}
+			JumpToCWord
```

Comment:
As JumpToCWord is a user supplied command, can you mention that
in comments?

Reply:
Changed the example a bit and made this user-defined part clearer.